uptane-standard.md

```diff
@@ -587,17 +598,17 @@ The inventory database MUST record the following pieces of information:
     * The vehicle identifier the ECU is associated with
     * A public key
     * The format of the public key
-    * Whether the ECU is a primary or a secondary
+    * Identification of the ECU as a primary or a secondary
 
 The inventory database MAY record other information about ECUs and vehicles.
 
 ### Time Server {#time_server}
 
-The Time Server exists to inform vehicles about the current time in cryptographically secure way, since many ECUs in a vehicle will not have a reliable source of time. It receives lists of tokens from vehicles, and returns back a signed sequence that includes the token and the current time.
+The Time Server exists to inform vehicles about the current time in a cryptographically secure way, since many ECUs in a vehicle will not have a reliable source of time. It receives lists of tokens from vehicles, and returns back a signed sequence that includes the token and the current time.
 
 The Time Server SHALL receive a sequence of tokens from a vehicle representing all of its ECUs. In response, it SHALL sign each token together with the current time.
 
-The Time Server SHALL expose a public interface allowing primaries to communicate with it. This communication MAY occur over FTP, FTPS, SFTP, HTTP, or HTTPS.
+The Time Server SHALL expose a public interface to allow primaries to communicate with it. This communication MAY occur over FTP, FTPS, SFTP, HTTP, or HTTPS.
```

Comment:
This should either stay as "allowing", or change to "that allows". "To allow" changes it from normative to explanatory.

Reply:
Good point. I'm willing to put this back.